contrib/report_builders/report_builder.py

```diff
@@ -8,80 +8,62 @@
 
 
 class ReportBuilder(metaclass=abc.ABCMeta):
-    @abc.abstractmethod
```

Comment:
Why did you decide to remove these?

Reply:
Those decorators obligate type to implement all methods, however, some report builder types (both json and html) do not need them. Most methods only useful if report internaly is built in sequential order, in that case, they still can be used, but it is not imperative now.